### DISCUSSION TITLE
Tree view for left side content navigation bar

### DISCUSSION THREAD

**Discussion by pchebro** [directus/directus#11389] [Archive]
Hi, 
I have a question regarding left side content navigation bar -  Is there a way to customize the below collection view as tree structure? For example, I would like to have Links and Documents appear as child pages inside Universal Navigation with collapse and expand button. Is there any reference documentation that you can point to? Please let me know if I need to open new feature request.

![image](https://user-images.githubusercontent.com/89423003/152153538-6e238ea8-40f3-4fa9-bf66-00f9fe709f9d.png)

**Comment by xu4wang** [comment]
You can create a folder, and put related models inside the folder.   

go to Settings -> Data Model, 
click the "folder with a +" icon at upper right conner, Note it's on the left of the big "+" icon for creating new model. 

You can customize the folder name, icon and models inside.

**Reply by pchebro** [reply]
Hi, That did the magic. Thank you so much.
